src/frontend/js/components/PaginateCourseSearch/index.tsx

```diff
@@ -123,7 +123,9 @@ export const PaginateCourseSearch = ({ courseSearchTotalCount }: PaginateCourseS
                 </li>
               ) : (
                 <li className="pagination__item">
-                  <a
```

Comment:
I feel like this might be wrong.

Technically, the interaction triggers a JS handler, which infringes on the lint rule.

In practice however, clicking on a pagination is semantically closer to switching pages than doing an "action" with a button. IMO this should be a link, not a button.

Reply:
Yep, in the case of pagination its closer but a `<a>` without `href` attribute is not something valid from W3C standard and I agree with your comment below, the best element for this case is a button. I will use a button instead of a span and update styles accordingly.

Reply:
> a <a> without href attribute is not something valid from W3C standard

Right, I forgot about this. I guess we'd need a router to make it a real link. Let's go with button then 👍